### ISSUE TITLE
PBMETAD possible issue with namd2 v2.13

### ISSUE THREAD

**Issue by ztimol** [plumed/plumed2#578]
Possible bug when attempting to run PBMETAD  in namd2-cuda v2.13 patched with plumed v2.6.

Tried to run the basic PBMETAD distance calculation from the docs and system crashed with atoms moving too fast error on the first time-step. In the colvar file bias reads 'nan'. 

This occurs both when trying to run a PBMETAD on 2 colvars and 1 colvar (if you want to do the latter with PBMETAD for some reason).

Basic METAD on 1 and 2 colvars runs without issue on same system. Other plumed functionality seems to run okay so don't think it's a compilation/patching issue. Also running system under NVT.

Haven't found anything directly related to this in the forum but did note this [pbmetad issue](https://groups.google.com/forum/embed/?place=forum/plumed-users&showsearch=true&showpopout=true&showtabs=false&parenturl=https%3A%2F%2Fwww.plumed.org%2F%2Fforum.html#!searchin/plumed-users/pbmetad/plumed-users/nzaBx8MjCQ4/WsS8SWNGAgAJ)

Plumed config file, plumed covlar file and namd logs attached. Hills file is empty.

[md_sol_plumed.log](https://github.com/plumed/plumed2/files/4739491/md_sol_plumed.log)
[plumed_metad_dist.txt](https://github.com/plumed/plumed2/files/4739495/plumed_metad_dist.txt)
[COLVAR.txt](https://github.com/plumed/plumed2/files/4739496/COLVAR.txt)

Not sure if any other info is required but if so let me know.

Thanks a lot

**Comment by GiovanniBussi** [comment]
Hi,

as far as I know, the NAMD patch distributed with PLUMED does not contain the commands necessary to implement replica exchange with different Hamiltonians, so I would say this is just not supported. I would be surprised if other multi replica algorithms work correctly, because they also seem not supported

Giovanni

**Comment by ztimol** [comment]
Hi,

Ah of course - I really should have realised that was the issue. Will port over to gromacs to give it a try at some point.

Thanks a lot for the help.

**Comment by carlocamilloni** [comment]
sorry maybe I missed something, but this looked like a single replica calculation, or was it actually replica-exchange? PBMetaD works in single replica

**Comment by GiovanniBussi** [comment]
Arg, my fault. I apologize, this is definitely to be checked.

**Comment by carlocamilloni** [comment]
anyway the issue is already at time zero when there should not be any force applied, can you try to  set the hills height to zero?

**Comment by maxbonomi** [comment]
Hi,

I have to admit PBMETAD might not have been thoroughly tested with standard metadynamics,
as it was conceived to be used mostly in combination with well-tempered metadynamics.
Could you try you switched on well-tempered metad by adding for example:

BIASFACTOR=8

to the PBMETAD line?

Max

**Comment by carlocamilloni** [comment]
I also thought the same, but I tested locally and it works, still it is worth checking

**Comment by ztimol** [comment]
_> sorry maybe I missed something, but this looked like a single replica calculation, or was it actually replica-exchange? PBMetaD works in single replica_


Apologies for the delayed response. I'm new to PLUMED and this form of analysis in general. Just wanted to make sure I didn't miss something on my part.

As you've already picked up it's a standard MD simulation (that is no tcl scripts were used to employ replica exchange). 

I did note this [Enhanced Conformational Sampling 2015 paper](https://pubs.acs.org/doi/10.1021/ct5009087) referenced in the docs. However, I assumed (likely incorrectly) that it would be possible to run PBMETAD in a standard MD simulation. Out of curiosity is it even possible (or even sensible) to run a multidimensional collective variable parallel bias metadynamics simulation without replica exchange?

In reality I don't need to employ PBMETAD and was just testing PLUMED functionality. i looked into it as I was actually having issues with both standard and well-tempered 2D metadynamics on ring puckering that I think I now may have managed to work around. 

I observed a strange artifact in the molecular structure. I'm just testing to see if it occurs again and that it wasn't an error on my part. If it is a possible issue will post it here in the next few days.

Thanks for the help and I'm happy to assist in any way I can.
